app/views/users/show.html.erb

```diff
@@ -1,2 +1,23 @@
-<h1>Here you will find a specific user. The page will display user name, number of posts, bio and a summary of the posts</h1>
+<div class="user-details">
+  <div class="user-container">
+    <div class="user-img black-border">
+      <%= image_tag("icon.png") %>
+    </div>
+    <div class="user-overview black-border">
+      <h2><%= @user.name %></h2>
+      <span>Number of posts: <%= @user.posts_counter || 0 %></span>
+    </div>
+  </div>
+  <div class="user-bio black-border">
+    <span>Bio</span>
+    <p><%= @user.bio %></p>
+  </div>
+  <% @user.posts.each do |post| %>
+    <div class="post-overview black-border">
+      <h3><%= link_to post.title, user_posts_path(post) %></h3>
```

Comment:
- [ ] Currently, when I click on this post title link, it takes me to post index instead of post show. According to the requirement, this link should take the user to see the details of the post the link was clicked instead of showing all the user's posts. You can easily resolve this by changing `<h3><%= link_to post.title, user_posts_path(post) %></h3>` to `<h3><%= link_to post.title, user_post_path(post) %></h3>`

Reply:
Thanks for pointing it out